content/docs/2.6/deploy.md

```diff
@@ -43,12 +43,21 @@ Deploying KEDA with Helm is very simple:
 
 ### Uninstall
 
-If you want to remove KEDA from a cluster you can run one of the following:
+If you want to remove KEDA from a cluster, you first need to remove any ScaledObjects that you have created. Once that is done, the Helm chart can be uninstalled:
```

Comment:
You have to remove ALL KEDA custom resources, ScaledObjects, ScaledJobs, TriggerAuthentication and ClusterTriggerAuthentication. 
Should we add all of them?
@zroubalik

Reply:
I didn't find that to be the case, @JorTurFer - although with that said, in my cluster I did only have TriggerAuthentication objects in addition to my ScaledObjects. The TriggerAuthentication resources I created after installing the Helm chart were cleanly removed when the Helm chart was uninstalled.

Reply:
...although on reflection, I suppose it does no harm to clean them out first anyway, I can update the PR accordingly

Reply:
you are right, not all resources have finalizers, only ScaledObject and ScaledJob, so we only need to add those 2